docs/src/main/paradox/project/downstream-upgrade-strategy.md

```diff
@@ -3,41 +3,36 @@ project.description: Upgrade strategy for downstream libraries
 ---
 # Downstream upgrade strategy
 
-When a new Akka version is released, downstream projects (such as
-[Akka Management](https://doc.akka.io/docs/akka-management/current/),
-[Akka HTTP](https://doc.akka.io/docs/akka-http/current/) and
-[Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/))
+When a new Pekko version is released, downstream projects (such as
+[Pekko Management](https://doc.akka.io/docs/akka-management/current/),
+[Pekko HTTP](https://doc.akka.io/docs/akka-http/current/) and
+[Pekko gRPC](https://doc.akka.io/docs/akka-grpc/current/))
 do not need to update immediately: because of our
 @ref[binary compatibility](../common/binary-compatibility-rules.md) approach,
-applications can take advantage of the latest version of Akka without having to
+applications can take advantage of the latest version of Pekko without having to
 wait for intermediate libraries to update.
 
 ## Patch versions
 
-When releasing a new patch version of Akka (e.g. 2.5.22), we typically don't
-immediately bump the Akka version in satellite projects.
+When releasing a new patch version of Pekko (e.g. 2.5.22), we typically don't
+immediately bump the Pekko version in satellite projects.
 
 The reason for this is this will make it more low-friction for users to update
-those satellite projects: say their project is on Akka 2.5.22 and
-Akka Management 1.0.0, and we release Akka Management 1.0.1 (still built with
-Akka 2.5.22) and Akka 2.5.23. They can safely update to Akka Management 1.0.1
-without also updating to Akka 2.5.23, or update to Akka 2.5.23 without updating
-to Akka Management 1.0.1.
+those satellite projects: say their project is on Pekko 2.5.22 and
```

Comment:
examples are still for versions that cannot exist here. Maybe we'll want to adapt this